src/components/dataset/entities.vue

```diff
@@ -11,7 +11,7 @@ except according to the terms contained in the LICENSE file.
 -->
 
 <template>
-    <div>
```

Comment:
Just noticed this while I was looking at this component! The `<div>` was indented a little too much.